homeassistant/components/homekit_controller/config_flow.py

```diff
@@ -115,15 +118,17 @@ def __init__(self) -> None:
         self.category: Categories | None = None
         self.devices: dict[str, AbstractDiscovery] = {}
         self.controller: Controller | None = None
-        self.finish_pairing: Awaitable[AbstractPairing] | None = None
+        self.finish_pairing: FinishPairing | None = None
```

Comment:
Maybe include the module in the strict typing list?

Reply:
Thats the end goal. It's a bit of a WIP already - we've already got a few hkc modules in the strict typing list. Config flow needs more work. These are just the ones needed to get pylint to pass without warnings locally.

Reply:
41163fc946e2c903a683c310349ed2ebf38fd5ed adds some more typing that gets us very close to including config_flow in the strict typing list - just 2 errors left. But they need fixes in aiohomekit. One is hard to do properly, so i'll do a follow up PR for those.